exercise_012_split_brain_resolver_static_quorum_http_mamagement/src/main/scala/org/neopixel/package.scala

```diff
@@ -0,0 +1,159 @@
+/**
+  * Copyright © 2018 Lightbend, Inc
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  *
+  * NO COMMERCIAL SUPPORT OR ANY OTHER FORM OF SUPPORT IS OFFERED ON
+  * THIS SOFTWARE BY LIGHTBEND, Inc.
+  *
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+package org
+
+import _root_.neopixel.{rpi_ws281x => ws}
+import _root_.neopixel.ws2811_t
+import _root_.neopixel.{rpi_ws281xConstants => wsC}
+import _root_.neopixel.{rpi_ws281xJNI => wsJ}
+import _root_.neopixel.ws2811_channel_t
+import _root_.neopixel.{ws2811_return_t => wsRet}
+
+import scala.collection.mutable.ListBuffer
+
+package object neopixel {
+
+  val Black = color(0, 0, 0)
+  val Green = color(255, 0, 0)
+  val MediumGreen = color(95, 0, 0)
+  val DarkGreen = color(60, 0, 0)
+  val Orange = color(255, 165, 0)
+  val DarkOrange = color(255, 140, 0)
+  val Red = color(0, 255, 0)
+  val DarkRed = color(0, 100, 0)
+  val Blue = color(0, 0, 255)
+  val LightBlue = color(40, 40, 255)
+  val DarkBlue = color(0, 0, 100)
+  val Yellow = color(255, 255, 0)
+  val Cyan = color(255, 0, 255)
+  val Magenta = color(0, 255, 255)
+  val White = color(255, 255, 255)
+  val WhiteLow = color(100, 100, 100)
+
+  val availableColorMap: Map[String, Long] =
+    Map(
+      "Black" -> Black,
+      "Green" -> Green,
+      "MediumGreen" -> MediumGreen,
+      "DarkGreen" -> DarkGreen,
+      "Orange" -> Orange,
+      "DarkOrange" -> DarkOrange,
+      "Red" -> Red,
+      "DarkRed" -> DarkRed,
+      "Blue" -> Blue,
+      "DarkBlue" -> DarkBlue,
+      "Yellow" -> Yellow,
+      "Cyan" -> Cyan,
+      "Magenta" -> Magenta,
+      "White" -> White,
+      "WhiteLow" -> WhiteLow
+    )
+
+  def color(red: Int, green: Int, blue: Int, white: Int = 0): Long =
+    (white << 24) | (red << 16)| (green << 8) | blue
+
+  case class LED_Data (channel: ws2811_channel_t, size: Int) {
+//    val ledData: Array[Long] = Array.fill[Long](size)(0L)
+    val ledData = ListBuffer.fill[Long](size)(0L)
+
+    def getitem(pos: Int): Long = ws.ws2811_led_get(channel, pos)
+
+    def setitem(pos: Int, value: Long) = {
+      ledData(pos) = value
+      ws.ws2811_led_set(channel, pos, value)
+    }
+  }
+
+  object Adafruit_NeoPixel {
```

Comment:
This seems to be duplicated a lot, perhaps move it to a library? (perhaps in a separate PR)

Reply:
Good point - To be marked in an issue.